CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.8.2 - 2023-03-17
+### Changed
+- Replies from `/findroom` are now sent ephemerally.
+
+## 0.8.0 - 2023-03-08
+### Added
+- `/findroom` command.
+
+## 0.7.1 - 2023-02-04
+### Changed
+- Docker is now optional.
+
```

Comment:
```suggestion
## 0.8.2 - 2023-03-17
### Changed
- Replies from `/findroom` are now sent ephemerally.

## 0.8.1 - 2023-03-16
### Added
- Custom errors types for `/update` command.

## 0.8.0 - 2023-03-08
### Added
- A new `/findroom` command for searching available rooms on campus.

## 0.7.1 - 2023-02-04
### Added
- GitHub workflow stages for export-version and linting.
- VS Code devcontainer setup.
- README sections on testing and devcontainers.

### Changed
- Updated node versioning across the project.
- Simplified npm scripts to be more intuitive.
- Docker is now optional.

### Fixed
- Bug where replied interactions couldn't report errors.
- Missing "git" dependency for /update command in Docker container.
```

Reply:
Let me know if I'm being _too_ detailed in the CHANGELOG. Not sure what level of changes are important enough to be reported.

Reply:
This is perfect. Userland ones might be most relevant, but since our audience is technical anyway, all of these details are interesting too!